lib/sandbox/pmapi.js

```diff
@@ -250,6 +251,26 @@ function Postman (execution, onRequest, onAssertion, cookieStore, options = {})
             });
 
             return self;
+        },
+
+        /**
+         * @interface ExecutionInterface
+         */
+        /**
+         * Exposes handlers to control or access execution state
+         *
+         * @type {ExecutionInterface}
+         */
+        execution: /** @lends ExecutionInterface */ {
+            /**
+             * Stops the current request from being sent and prevents any further
+             * scripts (from the current request) from being executed.
+             *
+             * @type {Function}
+             * @excludeFromTestScript
+             * @instance
+             */
+            skipRequest: onSkipRequest
```

Comment:
The autogenerated type is not accurate.

<img width="657" alt="image" src="https://github.com/postmanlabs/postman-sandbox/assets/24985700/22d622a5-945a-43d2-b551-7d41b0769c22">

Reply:
Corrected the type now